pipelines/toast_ground_schedule.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright (c) 2015-2020 by the parties listed in the AUTHORS file.
+# Copyright (c) 2015-2019 by the parties listed in the AUTHORS file.
```

Comment:
We should change that back to 2020

Reply:
thanks Ted. Yes, this one is tested thoroughly.

We don't have unit tests for the pipelines. Perhaps we should.